ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1155,18 +1155,19 @@ class PartCentredForceHolder {
 
     plugin_.PrepareToReportCollisions();
 
-    // The collisions are reported and stored into |currentCollisions| in
-    // OnCollisionEnter|Stay|Exit, which occurred while we yielded.
-    // Here, the |currentCollisions| are the collisions that occurred in the
-    // physics simulation, which is why we report them before calling
-    // |AdvanceTime|.
+    // The collisions are reported by the
+    // CollisionReporter.OnCollisionEnter|Stay events, which occurred while we
+    // yielded.
+    // Here, the |CollisionReporter.collisions| are the collisions that occurred
+    // in the physics simulation.
     foreach (Vessel vessel1 in
              FlightGlobals.Vessels.Where(v => !v.packed && is_manageable(v))) {
       if (plugin_.HasVessel(vessel1.id.ToString())) {
         if (vessel1.isEVA && (vessel1.evaController.OnALadder ||
                               is_clambering(vessel1.evaController))) {
           var vessel2 = vessel1.evaController.LadderPart?.vessel;
           if (vessel2 != null && !vessel2.packed && is_manageable(vessel2)) {
+            Log.Info("Reporting climbing a ladder");
```

Comment:
Remove?

Reply:
No, the provenance of the collision is useful information. The actual part-to-part collison report also logs on the C++ side (with IDs and part names, but without the added context), so it’s not like this makes logspam much worse.